lessons/configuration-dotnet.md

```diff
@@ -77,7 +77,69 @@ The *General settings*  section contains settings about the platform, debugging,
 
 ## 3. Adding custom application settings
 
-### 3.1. Using local.settings.json (Stace)
+Now that you know how to set the application settings in the portal, how can we use those values in your application?
+
+And how can you still debug your application locally without having access to those values in the portal?
+
+In the following steps you will
+
+* Create a function application
+* Retrieve a custom application setting
+* Set it locally for debugging purposes
+* Publish the setting to Azure
+
+### Steps
+
+1. In VSCode, create a new HTTP Trigger Function App with the following settings:
+   1. Location: *AzureFunctions.Configuration*
+   2. Language: *C#*
+   3. Template: *HttpTrigger*
+   4. Function name: *ReadingEnvironmentVariables*
+   5. Namespace: *AzureFunctionsUniversity.Demo.Configuration*  
+   6. AccessRights: *Function*
+2. Change the `FunctionName` attribute to reflect the name of the class
+```c#
+[FunctionName(nameof(ReadingEnvironmentVariables))]
+```
+3. Remove the code inside the `Run` function and replace with the following snippet
+```c#
+log.LogInformation("ReadingEnvironmentVariables Triggered via HTTP");
+var config = Environment.GetEnvironmentVariable("ConfigurationValue");
+return new OkObjectResult($"ConfigurationValue: {config}");
+```
+4. The finished function should look something like
+```c#
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace AzureFunctionsUniversity.Demo.Configuration
+{
+    public static class ReadingEnvironmentVariables
+    {
+        [FunctionName(nameof(ReadingEnvironmentVariables))]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("ReadingEnvironmentVariables Triggered via HTTP");
```

Comment:
I suggest to remove the logging statement. Doesn't add any value here.

Reply:
Removed!